_posts/2022-04-30-keeping-up-with-carpentrycon2022.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "Keeping up with CarpentryCon2022"
+date:   2022-04-30 09:00:00
+isStaticPost: false
+tags: announcements
+image: cc2020-pebbles.jpg
+---
+
+CarpentryCon news and updates will be circulated on various channels. Here's a quick guide on dates to save and sources to watch. 
+We're very excited for the conference, and details about proposal submission, keynote speakers, and registration will be available soon! 
+
+Stay tuned to the following channels to hear about our keynote speakers and other updates:
+
+- [CC@Home on Twitter](https://twitter.com/carpentrycon)
```

Comment:
I thought we were moving toward CarpentryCon 2022 as opposed to CC@Home...

Reply:
Yes. Missed those. I'll update all of these and add.